src/SFA.DAS.FAT.Domain/Courses/Provider.cs

```diff
@@ -69,8 +69,14 @@ public class FeedbackDetail
 
     public class FeedbackAttributes
     {
-        public List<string> Strengths { get; set; }
-        public List<string> Weaknesses { get; set; }
+        public List<FeedbackAttributeDetail> FeedbackAttributeDetail { get; set; }
```

Comment:
This may change if we simplify the API side to reduce nesting

Reply:
Done